modules/central-state/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@phenyl/central-state",
+  "version": "1.0.0",
+  "description": "",
+  "repository": "phenyl-js/phenyl",
+  "license": "Apache License 2.0",
+  "author": "Shin Suzuki <shinout310@gmail.com>, Nicholas Lee <nicholaslee119@gmail.com>",
+  "files": [
+    "lib",
+    "LICENSE",
+    "README.md"
+  ],
+  "main": "lib/index.js",
+  "types": "lib/index.d.js",
```

Comment:
types should be changed to `lib/index.d.ts`

Reply:
Fixed!